WELCOME.md

```diff
@@ -0,0 +1,47 @@
+👋 Hello @new_user_name and welcome to the For the Horde community! We're so happy you're here! The purpose of this community is to give you a space to connect with other Minnesota-based tech and tech-adjacent folks. Want to be successful here? Give more than you take.
+
+✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to [http://git.io/v0d6b](http://git.io/v0d6b).
```

Comment:
```suggestion
✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to [http://git.io/v0d6b](http://git.io/v0d6b) to check it out.
```

Reply:
I also wonder if we should require people to check a box, like in a google form. Not that we have to check it all the time, but to have a record.

Thoughts?

Reply:
That's a cool idea! I was overengineering and asking myself if I could eventually write a slack app that would deliver this message and have an interactive element for the user to click to record acknowledgement, but a google form is awesome because it gets the job done now.